samples/browser/src/main/java/org/mozilla/samples/browser/BrowserFragment.kt

```diff
@@ -46,6 +51,13 @@ class BrowserFragment : Fragment(), BackHandler {
                 sessionId)
 
         tabsToolbarFeature = TabsToolbarFeature(context!!, toolbar, ::showTabs)
+
+        val ctx = context!!.applicationContext
```

Comment:
* nit: You can use `requireContext` in Fragments if you want to avoid `context!!` and are expecting to always have. a `Context`.
* nit: I feel the shorter `ctx` is harder to skim then just using `context` or `applicationContext` :)

Reply:
Nice I didn't know about `requireContext()`, thanks for the suggestion!